src/html.pre.js

```diff
@@ -48,9 +50,29 @@ function pre(context) {
   }
 
   // ensure content.data is present
-  if (!context.content.data) {
-    context.content.data = {};
+  if (!content.data) {
+    content.data = {};
   }
```

Comment:
Nitpicking, but shorter:
```
content.data = content.data || {};
```

Reply:
Not my code, but I'll gladly enhance it 😄